packages/sync/src/modules/Callables.php

```diff
@@ -237,6 +237,25 @@ public function enqueue_full_sync_actions( $config, $max_items_to_enqueue, $stat
 		return array( 1, true );
 	}
 
+	/**
+	 * Send the callable actions for full sync.
+	 *
+	 * @access public
+	 *
+	 * @param array   $config Full sync configuration for this sync module.
+	 * @param int     $send_until Maximum duration of processing.
+	 * @param boolean $state True if full sync has finished enqueueing this module, false otherwise.
+	 *
+	 * @return array Number of actions enqueued, and next module state.
+	 */
+	public function send_full_sync_actions( $config, $send_until, $state ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		// we call this instead of do_action when sending immediately.
+		$this->send_action( 'jetpack_full_sync_callables', [ true ] );
+
+		// The number of actions enqueued, and next module state (true == done).
+		return array( 1, true );
```

Comment:
For consistency with other changes, this should be `return [ 1, true ];`

Reply:
https://github.com/Automattic/jetpack/pull/13963/commits/cec3e26369344dcff25736e1f1007c86a68800d8